website/docs/r/api_management.html.markdown

```diff
@@ -162,6 +162,10 @@ A `management`, `portal`, `developer_portal` and `scm` block supports the follow
 
 * `negotiate_client_certificate` - (Optional) Should Client Certificate Negotiation be enabled for this Hostname? Defaults to `false`.
 
+* `identity_client_id` - (Optional) The client id of the System or User Assigned Managed identity generated by Azure AD, which has `GET` access to the keyVault containing the SSL certificate.
```

Comment:
is this JUSt used for the cert and keyvault? should this be 
```suggestion
* `ssl_keyvault_identity_client_id` - (Optional) The client id of the System or User Assigned Managed identity generated by Azure AD, which has `GET` access to the keyVault containing the SSL certificate.
```
to be explicit what the identity is being used for?